packages/plugin-dev/src/browser/hosted-plugin-manager-client.ts

```diff
@@ -298,7 +298,7 @@ export class HostedPluginManagerClient {
 
         if (this.pluginInstanceURL) {
             try {
-                this.windowService.openNewWindow(this.pluginInstanceURL);
+                this.windowService.openNewWindow(this.pluginInstanceURL, { windowName: 'hosted-theia', opener: true });
```

Comment:
open new window cannot be used to reuse the window, you should use window.locaiton.href instead to navigate to another location

Reply:
I didn't understand what should i do with `href`, i already have `pluginInstanceUrl`.

If i can't use `openNewWindow`, can i use browser's window.open? 
What is the impact on electron?

Reply:
> I didn't understand what should i do with href, i already have pluginInstanceUrl.

`window.location.href = pluginInstanceUrl.toString(false)`;

> If i can't use openNewWindow, can i use browser's window.open?
What is the impact on electron?

api changes don't look good, `opener` is an implementation details of browser implement of `WindowService` and should not be exposed.